store/postgres.go

```diff
@@ -319,24 +400,19 @@ func psqlTableCreate(tenant string, table core.Table) (string, error) {
 			return "", fmt.Errorf("failed to create SQL statement for table: %s: %w", table.Name, err)
 		}
 		tableFields = append(tableFields, field.Name+" "+sqlType)
-
-		if field.Unique {
-			uniqueFields = append(uniqueFields, field.Name)
-		}
 	}
 	// Add the joins as fields to the SQL table
 	for _, join := range table.Joins {
-		tableFields = append(tableFields, join.Table+"_id SERIAL")
-	}
-
-	if len(uniqueFields) > 0 {
-		tableFields = append(tableFields, "UNIQUE ("+strings.Join(uniqueFields, ",")+")")
+		fieldName := join.Table + "_id"
+		tableFields = append(tableFields, fieldName+" INT8")
 	}
 
 	return "CREATE TABLE IF NOT EXISTS " + psqlAbsTableName(tenant, table.Name) + " ( " + strings.Join(tableFields, ",") + " );", nil
 }
 
-func psqlDataNodeUpsert(tenant string, node *dataNode, table core.Table) (sq.InsertBuilder, error) {
+// psqlDataCreateUpdate generates a sql query for performing an insert/update
```

Comment:
Curious why we don't use `psqlDataUpsert`, and equally set the policy type to `UpsertPolicy`?